core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -40,11 +40,11 @@ export class AudioBarComponent {
   isPaused: boolean = true;
   directiveSubscriptions: Subscription = new Subscription();
   languagesInExploration: ExplorationLanguageInfo[];
-  audioBarIsExpanded: boolean;
-  progressBarIsShown: boolean;
+  audioBarIsExpanded!: boolean;
+  progressBarIsShown!: boolean;
```

Comment:
Don't use null-assertion here instead initialize booleans with false, undefined for booleans does also mean that they are false

Reply:
Hi! Just pushed a change that fixes this one.